conditionals.py

```diff
@@ -210,4 +210,22 @@ def exclude_mapcompass_info_remove(random_settings, weight_dict, **kwargs):
 def ohko_starts_with_nayrus(random_settings, weight_dict, extra_starting_items, **kwargs):
     """ If one hit ko is enabled, add Nayru's Love to the starting items """
     if random_settings['damage_multiplier'] == 'ohko':
-        extra_starting_items['starting_items'] += ['nayrus_love']
\ No newline at end of file
+        extra_starting_items['starting_items'] += ['nayrus_love']
+
+def invert_dungeons_mq_count(random_settings, weight_dict, **kwargs):
+    """ When activated will invert the MQ dungeons count
+        kwargs: [chance of having the MQ count reversed]
+    """
+    if random_settings['mq_dungeons_mode'] != 'count':
+        return
+
+    chance_of_inverting_mq_count = int(kwargs['cparams'][0])
+    invert_mq_count = random.choices([True, False], weights=[chance_of_inverting_mq_count, 100-chance_of_inverting_mq_count])[0]
+
+    if not invert_mq_count:
+        return
+    
+    current_mq_dungeons_count = int(random_settings['mq_dungeons_count'])
+    new_mq_dungeons_count = 12 - current_mq_dungeons_count
+    
+    random_settings['mq_dungeons_count'] = str(new_mq_dungeons_count)
```

Comment:
Is there a reason you converted this to a string? Might be better to just leave it as an int. I also don't remember if `12-int(number)` guarantee that you get an int back? Either way, doesn't hurt to be safe if you just change the `str` to `int` just to be sure.

Reply:
I thought the original value was stored as a string but it might be a confusion from me, I'm gonna test without it

Reply:
done